frameworks/hdfs/universe/marathon.json.mustache

```diff
@@ -85,9 +86,11 @@
 
 
     {{#service.kerberos.enabled}}
-    "KEYTABS_URI": "{{service.kerberos.keytabs_uri}}",
-    "KRB5_CONF_URI": "{{service.kerberos.krb5_conf_uri}}",
-    "TASKCFG_ALL_SECURE_MODE": "{{service.kerberos.enabled}}",
+    "KERBEROS_ENABLED": "{{service.kerberos.enabled}}",
```

Comment:
Are both of these required?

Reply:
So `TASKCFG_ALL_*` isn't generally used in the service spec, since any env vars that are prefixed with this are meant for tasks.

In this case, `KERBEROS_ENABLED` is used in the service spec, and `TASKCFG_ALL_KERBEROS_ENABLED` is used to properly render template configs on the tasks' environments.

Same things applies for TLS configs.

Reply:
Thanks for the clarification.

Reply:
There is no real moral purity around not using TASKCFG_ALL_* in svc.yml. We have done it both ways.

Arguably using just the TASKCFG_ALL makes it slightly easier to maintain.